### PR TITLE
JTD-005 Create Delivery Employee Table

### PR DESCRIPTION
**Pull request by james-liam-butler**

### CODE REVIEW

File: src/main/resources/db/migration/V5__create_delivery_employee_table.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+CREATE TABLE deliveryEmployee (
+	delEmpId int NOT NULL AUTO_INCREMENT,
+    `name` varchar(50),
+    salary decimal(7, 2),
+    currency char(3),
+	bankAccountNumber varchar(30),
+    nationalInsurance varchar(30),
+    PRIMARY KEY(delEmpId)
```
Comment (lukemKainos) [inline src/main/resources/db/migration/V5__create_delivery_employee_table.sql]: fix the formatting on this please
How long is a national insurance number? Do you need 30?
Whats delEmpId

**Review by lukemKainos — CHANGES_REQUESTED** [review]

File: src/main/resources/db/migration/V5__create_delivery_employee_table.sql [hunk 2]
```diff
@@ -0,0 +1,9 @@
+CREATE TABLE deliveryEmployee (
+	deliveryEmployeeId int NOT NULL AUTO_INCREMENT,
+	`name` varchar(50),
+	salary decimal(7, 2),
+	currency char(3),
+	bankAccountNumber varchar(34),
+	nationalInsurance varchar(9),
+	PRIMARY KEY(delEmpId)
```
Comment (lukemKainos) [inline src/main/resources/db/migration/V5__create_delivery_employee_table.sql]: what is delEmpId?
This hasnt been tested

File: src/main/resources/db/migration/V5__create_delivery_employee_table.sql (hunk 1 above)
Comment (james-liam-butler) [reply src/main/resources/db/migration/V5__create_delivery_employee_table.sql]: fixed

File: src/main/resources/db/migration/V5__create_delivery_employee_table.sql [hunk 3]
```diff
@@ -0,0 +1,9 @@
+CREATE TABLE deliveryEmployee (
+	deliveryEmployeeId int NOT NULL AUTO_INCREMENT,
+	`name` varchar(50),
+	salary decimal(7, 2),
+	currency char(3),
+	bankAccountNumber varchar(34),
+	nationalInsurance varchar(9),
```
Comment (lukemKainos) [inline src/main/resources/db/migration/V5__create_delivery_employee_table.sql]: national insurance what?

**Review by lukemKainos — CHANGES_REQUESTED** [review]

File: src/main/resources/db/migration/V5__create_delivery_employee_table.sql (hunk 3 above)
Comment (james-liam-butler) [reply src/main/resources/db/migration/V5__create_delivery_employee_table.sql]: Number

**Review by lukemKainos — APPROVED** [review]

**Review by dana-kainos — APPROVED** [review]

**Pull request merged**